hypervisor/hw/pci.c

```diff
@@ -61,6 +61,81 @@ uint64_t get_mmcfg_base(void)
 	return pci_mmcfg_base;
 }
 
+#if defined(HV_DEBUG)
+static inline uint32_t pci_off_to_address(union pci_bdf bdf, uint32_t offset)
```

Comment:
pio_off_to_address?

Reply:
Also for pci_mmcfg_calc_address, please rename similarly?

Reply:
done